SECURITY.md

```diff
@@ -0,0 +1,79 @@
+# Security Policy
+
+This policy is adapted from the policies of the following CNCF projects:
+
+* [Rook](https://github.com/rook/rook)
+* [Containerd](https://github.com/containerd/project)
+
+## Reporting a Vulnerability
+
+To report a vulnerability, either:
+
+1. Report it on Github directly you can follow the procedure described
+   [here](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability)
+   and:
+    - Navigate to the [security tab](https://github.com/crossplane/crossplane/security) on the repository
+    - Click on 'Advisories'
+    - Click on 'Report a vulnerability'
+
+2. Send an email to `crossplane-security@lists.cncf.io` detailing the issue and steps
+to reproduce.
+
+The reporter(s) can typically expect a response within 24 hours acknowledging
+the issue was received. If a response is not received within 24 hours, please
+reach out to any
+[maintainer](https://github.com/crossplane/crossplane/blob/master/OWNERS.md#maintainers)
+directly to confirm receipt of the issue.
+
+## Review Process
+
+Once a maintainer has confirmed the relevance of the report, a draft security
+advisory will be created on Github. The draft advisory will be used to discuss
+the issue with maintainers, the reporter(s), and crossplane's security advisors.
+If the reporter(s) wishes to participate in this discussion, then provide
+reporter Github username(s) to be invited to the discussion. If the reporter(s)
+does not wish to participate directly in the discussion, then the reporter(s)
+can request to be updated regularly via email.
+
+If the vulnerability is accepted, a timeline for developing a patch, public
+disclosure, and patch release will be determined. If there is an embargo period
+on public disclosure before the patch release, an announcment will be sent to
+the security announce mailing list (`crossplane-security-announce@lists.cncf.io`)
+announcing the scope of the vulnerability, the date of availability of the
+patch release, and the date of public disclosure. The reporter(s) are expected
+to participate in the discussion of the timeline and abide by agreed upon dates
+for public disclosure.
+
+## Public Disclosure Process
```

Comment:
I would recommend adding a line to urge reporters to not discuss issues before Crossplane have announced them.

Reply:
Wait, it is already added in the previous section.